src/Resources/ContactProperties.php

```diff
@@ -16,7 +16,7 @@ class ContactProperties extends Resource
      */
     public function all()
     {
-        $endpoint = 'https://api.hubapi.com/contacts/v2/properties';
+        $endpoint = "https://api.hubapi.com/contacts/v2/properties";
```

Comment:
Is it really necessary ?
You don’t use any params ...
it have to be wrapped ' (a single quotes)